src/keylessBackup/KeylessBackupIntro.tsx

```diff
@@ -1,69 +1,97 @@
+        {isSetup && <Text style={styles.title}>{t('keylessBackupIntro.setup.title')}</Text>}
+        <Text
+          style={[
+            isSetup ? { ...typeScale.bodyMedium } : { ...typeScale.labelMedium },
+            styles.description,
+          ]}
+        >
+          {isSetup
+            ? t('keylessBackupIntro.setup.description')
+            : t('keylessBackupIntro.restore.description')}
+        </Text>
         <Card style={styles.authFactorsCard} shadow={null}>
           <View style={styles.authFactorsContainer}>
             <View style={styles.authFactorLine}>
               <EnvelopeIcon style={styles.envelopeIcon} />
-              <Text style={styles.authFactorText}>{t('setUpKeylessBackup.emailAddress')}</Text>
+              <Text style={styles.authFactorText}>{t('keylessBackupIntro.emailAddress')}</Text>
             </View>
             <View style={styles.authFactorLine}>
               <SmartphoneIcon style={styles.smartphoneIcon} />
-              <Text style={styles.authFactorText}>{t('setUpKeylessBackup.phoneNumber')}</Text>
+              <Text style={styles.authFactorText}>{t('keylessBackupIntro.phoneNumber')}</Text>
             </View>
           </View>
 
           <Text style={styles.reminderText}>
-            <Trans i18nKey="setupKeylessBackupReminder">
-              {/* namespaced keys did not work for this */}
-              <Text style={styles.reminderPrefix} /> {/* prefix string gets injected here */}
+            <Trans
+              i18nKey={
+                isSetup
+                  ? 'keylessBackupIntro.setup.reminder'
+                  : 'keylessBackupIntro.restore.reminder'
+              }
+            >
+              <Text style={styles.reminderPrefix} />
             </Trans>
           </Text>
         </Card>
-        <TextButton
-          testID="SetUpKeylessBackup/RecoveryPhrase"
-          style={styles.recoveryPhrase}
-          onPress={onPressRecoveryPhrase}
-        >
-          {t('setUpKeylessBackup.useRecoveryPhrase')}
-        </TextButton>
+        {isSetup && (
+          <TextButton
+            testID="keylessBackupIntro/RecoveryPhrase"
+            style={styles.recoveryPhrase}
+            onPress={onPressRecoveryPhrase}
+          >
+            {t('keylessBackupIntro.setup.useRecoveryPhrase')}
+          </TextButton>
+        )}
       </ScrollView>
       <Button
-        testID="SetUpKeylessBackup/Continue"
-        onPress={onPressContinue}
-        text={t('continue')}
+        testID="keylessBackupIntro/Continue"
+        onPress={() => {
+          if (isSetup) {
+            ValoraAnalytics.track(KeylessBackupEvents.cab_setup_continue)
+            navigate(Screens.SignInWithEmail, { keylessBackupFlow: KeylessBackupFlow.Setup })
+          } else {
+            ValoraAnalytics.track(KeylessBackupEvents.cab_restore_continue)
+            navigate(Screens.SignInWithEmail, { keylessBackupFlow: KeylessBackupFlow.Restore })
+          }
```

Comment:
nit: we could use the same event for both and avoid the if else here
```suggestion
          ValoraAnalytics.track(KeylessBackupEvents.cab_intro_continue, { keylessBackupFlow })
          navigate(Screens.SignInWithEmail, { keylessBackupFlow })
```

Reply:
Changed to use one analytic event in ee39204f0a845d6f9420bfc5e935c901ce76bab1!